### PR TITLE
Port breach alert email to new email template

### PR DESCRIPTION
**Pull request by Vinnl**
# References:

Jira: MNTOR-3448
Figma: N/A

# Description

This updates the breach alert email to be based on the new email template (behind a flag).

# Screenshot (if applicable)

As received on Mailinator:

image

Previous version, for comparison:

image

# How to test

## Easiest

The email is present in Storybook, so you can [view it there](https://deploy-preview-4895--fx-monitor-storybook.netlify.app/?path=/story/emails-breach-alert--breach-alert-email-story).

## Doable
You can also send it to yourself via `/admin/emails`.

## Hard
However, if you want to actually verify that it works as it would do in production, that's a bit more involved.

The first step would be to enable the `RedesignedEmails` feature flag for your user. Then follow the instructions in the README under "PubSub". Then to make sure the email gets sent to your email addess, you'll want to modify the `getSubscribersByHashes` function in `src/db/tables/subscribers.js` to return your specific subscriber:

```diff
-    // .whereIn('primary_sha1', hashes)
+    .where("primary_email", "youraccount@mailinator.com")
```

If you then run the cronjob (`NODE_ENV="development" npm run dev:cron:breach-alerts`), the email should get sent to the address you specified.

If you want to repeat the above, you'll have to first run this query: `DELETE * FROM email_notifications`
Then, run the `curl` command again, and after that run the cronjob. The email should now get resent.

# Checklist (Definition of Done)

- [x] Localization strings (if needed) have been added.
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [x] I've added or updated the relevant sections in readme and/or code comments
- [x] I've added a unit test to test for potential regressions of this bug.
- [x] If this PR implements a feature flag or experimentation, the Ship Behind Feature Flag status in Jira has been set
- [ ] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [x] All acceptance criteria are met.
- [x] Jira ticket has been updated (if needed) to match changes made during the development process.
- [x] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage.

### CODE REVIEW

**Review by flodolo — APPROVED** [review]

File: src/app/(proper_react)/(redesign)/(authenticated)/admin/emails/EmailTrigger.tsx [hunk 1]
```diff
@@ -23,6 +24,7 @@ export const EmailTrigger = (props: Props) => {
     props.emailAddresses[0],
   );
   const [isSendingVerification, setIssSendingVerification] = useState(false);
+  const [isSendingBreachAlert, setIssSendingBreachAlert] = useState(false);
```
Comment (rhelmer) [inline src/app/(proper_react)/(redesign)/(authenticated)/admin/emails/EmailTrigger.tsx:27]:
```suggestion
  const [isSendingBreachAlert, setIsSendingBreachAlert] = useState(false);
```

Should this be `setIsSendingBreachAlert`?

File: src/app/(proper_react)/(redesign)/(authenticated)/admin/emails/EmailTrigger.tsx [hunk 2]
```diff
@@ -80,6 +82,18 @@ export const EmailTrigger = (props: Props) => {
         >
           Monthly activity overview
         </Button>
+        <Button
+          variant="primary"
+          isLoading={isSendingBreachAlert}
+          onPress={() => {
+            setIssSendingBreachAlert(true);
```
Comment (rhelmer) [inline src/app/(proper_react)/(redesign)/(authenticated)/admin/emails/EmailTrigger.tsx:89]:
```suggestion
            setIsSendingBreachAlert(true);
```

File: src/app/(proper_react)/(redesign)/(authenticated)/admin/emails/EmailTrigger.tsx [hunk 3]
```diff
@@ -80,6 +82,18 @@ export const EmailTrigger = (props: Props) => {
...
+            void triggerBreachAlert(selectedEmailAddress).then(() => {
+              setIssSendingBreachAlert(false);
```
Comment (rhelmer) [inline src/app/(proper_react)/(redesign)/(authenticated)/admin/emails/EmailTrigger.tsx:91]:
```suggestion
              setIsSendingBreachAlert(false);
```

**Review by rhelmer — CHANGES_REQUESTED** [review]
lgtm overall, noticed what might be a small typo in a variable name (it's consistent though so not causing any runtime errors)

File: locales/en/email-strings.ftl [hunk 4]
```diff
@@ -82,11 +82,14 @@ email-verify-subhead = Verify your email to start protecting your data after a b
 email-verify-simply-click = Simply click the link below to finish verifying your account.
 
 ## Breach report
-## Variables:
-##   $email-address (string) - Email address
 
 email-breach-summary = Here’s your data breach summary
+# Variables:
+#   $email-address (string) - Email address, bolded
```
Comment (flozia) [inline locales/en/email-strings.ftl:88]: Nit: The “bolded” string is the `email-breach-detected-2` one below, right?

**Review by flozia — APPROVED** [review]
Looks good!

File: src/app/(proper_react)/(redesign)/(authenticated)/admin/emails/EmailTrigger.tsx (hunk 1 above)
Comment (Vinnl) [reply src/app/(proper_react)/(redesign)/(authenticated)/admin/emails/EmailTrigger.tsx]: Haha, looks like that wasn't the only variable I made that typo for: 26e430156

File: locales/en/email-strings.ftl (hunk 4 above)
Comment (Vinnl) [reply locales/en/email-strings.ftl:88]: The original one included the `<b>` inside the variable (which Fluent-React doesn't allow us to do), whereas the new one gives the localiser more freedom to bold the relevant text (and works with Fluent-React).

**Review by rhelmer — APPROVED** [review]
lgtm, so nice to be able to see these in Storybook

**Pull request merged**